lib/lambda_ethereum_consensus/p2p/gossip/handler.ex

```diff
@@ -31,7 +39,7 @@ defmodule LambdaEthereumConsensus.P2P.Gossip.Handler do
     root = aggregate.data.beacon_block_root |> Base.encode16()
 
     # We are getting ~500 attestations in half a second. This is overwhelming the store GenServer at the moment.
-    # Store.on_attestation(aggregate)
+    # ForkChoice.on_attestation(aggregate)
```

Comment:
Oh, I didn't know this was still commented out. Created: https://github.com/lambdaclass/lambda_ethereum_consensus/issues/924